modules/displayScreen.js

```diff
@@ -0,0 +1,32 @@
+// onclick Lis
+/* eslint-disable no-unused-vars */
+function displayList() {
+  document.getElementById('book-table').style.display = 'flex';
+  document.getElementById('new-book').style.display = 'none';
+  document.getElementById('contact').style.display = 'none';
+  document.getElementById('dList').style.color = '#04aa6d';
+  document.getElementById('dAddBook').style.color = 'black';
+  document.getElementById('dContact').style.color = 'black';
+}
+
+// onclick Add Book
+
+function displayAddBook() {
+  document.getElementById('book-table').style.display = 'none';
+  document.getElementById('new-book').style.display = 'flex';
+  document.getElementById('contact').style.display = 'none';
+  document.getElementById('dList').style.color = 'black';
+  document.getElementById('dAddBook').style.color = '#04aa6d';
+  document.getElementById('dContact').style.color = 'black';
+}
+
+// onclick Contact
+
+function displayContact() {
+  document.getElementById('book-table').style.display = 'none';
+  document.getElementById('new-book').style.display = 'none';
+  document.getElementById('contact').style.display = 'block';
+  document.getElementById('dList').style.color = 'black';
+  document.getElementById('dAddBook').style.color = 'black';
+  document.getElementById('dContact').style.color = '#04aa6d';
+}
```

Comment:
- [ ] Let's make sure we use the arrow function instead, since this project is practice to ES6 module syntax

Reply:
Hello @Amrhub , Thank you for reviewing my code.
I have some trouble understanding how can I change the conde to make it function properly. 
As I'm calling the function inside the index.html with an onclick method 
` <li><p id="dList" onclick="displayList()">List</p></li>`
`<li><p id="dAddBook" onclick="displayAddBook()">Add book</p></li>`
`<li><p id="dContact" onclick="displayContact()">Contact</p></li>`

If I convert those functions to arrow functions, how can I call the functions in the HTML code?